tests/test_integration_org_unit.py

```diff
@@ -22,13 +22,14 @@ class Tests(util.LoRATestCase):
 
     def test_org_unit_temporality(self):
         self.load_sample_structures()
-
         self.assertRequestResponse(
             '/service/ou/04c78fc2-72d2-4d02-b55f-807af19eac48'
             '/details/org_unit?validity=past',
             [
                 {
                     "name": "Afdeling for Fremtidshistorik",
+                    'path': ('/Overordnet Enhed/Humanistisk fakultet/' +
```

Comment:
In the original customer request, the path didn't start with a `/`, did it? Also, the path doesn't include the name of the unit itself — is this intentional?

Reply:
I actually originally returned the answer without the starting slash:
https://github.com/magenta-aps/mora/commit/9e7510b5bf849c05968d79291bdea67e7465da96#diff-40eda68fe139c8fe75670d0531e68967R215
But then it was requested that the user_key should still be returned by the rest-interface and then I forgot about it. I will get rid of it again (and update all tests....)